### PR TITLE
build: add instructions to point webui to cors disabled clusters

### PR DESCRIPTION
**Pull request by hamidzr**
## Test Plan

- use the instruction to point `make live` dev server to a remote cluster

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ This is set to `/det` by default for typical workflows in this project.  [More i
 - `DET_NODE_ENV`: set this to `development` to skip some build optimizations when developing and building
 locally to lower build time.
 
+## Developing Against a Remote Cluster
+
+If the remote cluster has `enable_cors` set or allows CORS requests set `SERVER_ADDRESS` to point
+to the cluster address. If that's not he case use the bundled `local-cors-proxy` package to run a
```
Comment (apizzini) [inline webui/react/README.md:52]: typo: If that's not THE case

File: webui/react/README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ This is set to `/det` by default for typical workflows in this project.  [More i
 - `DET_NODE_ENV`: set this to `development` to skip some build optimizations when developing and building
 locally to lower build time.
 
+## Developing Against a Remote Cluster
+
+If the remote cluster has `enable_cors` set or allows CORS requests set `SERVER_ADDRESS` to point
```
Comment (apizzini) [inline webui/react/README.md:51]: took me a while to understand this paragraph (maybe my bad english):
- `has 'enable_cors' set` to any value?
- maybe put a comma `allows CORS requests, set ...`

**Review by apizzini — APPROVED** [review]
LGTM aside a possible typo and probably me not getting a simple sentence.

File: webui/react/README.md (hunk 2 above)
Comment (hamidzr) [reply webui/react/README.md:51]: both good points, thanks

**Pull request merged**